feature-libs/checkout/scheduled-replenishment/components/services/checkout-replenishment-form-service.ts

```diff
@@ -1,15 +1,34 @@
-import { Injectable } from '@angular/core';
```

Comment:
The file name should end with `.service.ts`, and not `-service.ts`

Reply:
oufff nice catch